meb/core/train_eval.py

```diff
@@ -332,7 +340,7 @@ def validate_split(
         self.setup_training()
 
         # Train and evaluation
-        self.train_model(train_loader, test_loader)
+        self.train_model(train_loader, test_loader, split_name)

```

Comment:
Will this break anything?

Reply:
No, because the parameter is used as a name for weights. no logic involved. it's actually a parameter that gets passed around in other validation-related function